users/api_test.py

```diff
@@ -35,6 +35,17 @@ def test_fetch_user(prop, value, db_value):
     assert user == found_user
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize("prop,value", [["email", "abc@example.com"]])
```

Comment:
If we only have one set of values here does it make sense to inline the values instead?

Reply:
Yeah, initially I also had username as case-insensitive-enabled before I realized it didn't really make sense. I can change the params